client/app/kubernetes/kubernetes.service.js

```diff
@@ -113,4 +113,12 @@ angular.module('managerApp').service('Kubernetes', class Kubernetes {
         return res;
       });
   }
+
+  updateKubernetesVersion(serviceName) {
+    return this.OvhApiKube.v6().updateVersion({ serviceName }, {}).$promise
+      .then((res) => {
```

Comment:
No error handling ?

Reply:
The error is being handled in the controller :): https://github.com/ovh-ux/ovh-manager-cloud/pull/1317/files#diff-e5e37356e49f3a7d7f69ba3d3b865957R28